### DISCUSSION TITLE
Does anyone know of any good ways to view Repositories without having to use Chrome Extensions?

### DISCUSSION THREAD

**Discussion by CuterNatives** [community/community#35246] [Other Feature Feedback, Questions, & Ideas] [answered]
Thanks for helping!

**Comment by Anirban-Saha** [comment, marked as answer]
You can use third-party extensions like Octotree and likes.
If that's not an option then replace github in the url with github1s.
Either of the solutions are good but I wouldn't recommend using them fully as I haven't used them personally yet.
Hope that helps!
